src/shared/lib/StringUtils.ts

```diff
@@ -29,4 +29,31 @@ export function stringListToIndexSet(alos:ReadonlyArray<string>):{[s:string]:num
         map[next] = index;
         return map;
     }, {});
+}
+
+/**
+ * Unquote a string wrapped with single or double quotes.
+ *
+ * @param {string} str  input string wrapped with single or double quotes
+ */
+export function unquote(str: string): string
+{
+    if ((str.startsWith('"') && str.endsWith('"')) ||
```

Comment:
startsWith/endsWith not supported in IE11.  Possibly already polyfilled by babel but please check to make sure.

Reply:
good point